bdk_esplora/src/ureq.rs

```diff
@@ -188,17 +180,32 @@ impl Client {
         Ok(())
     }
 
+    pub fn spk_scan(
+        &self,
+        spks: impl Iterator<Item = Script>,
+        existing_chain: BTreeMap<u32, BlockHash>,
+    ) -> Result<ChainGraph<ConfirmationTime>, UpdateError> {
```

Comment:
General question regarding this esplora module.. Will this api eventually be upstream 'd into `rust-esplora` library? I remember we had a team call where we discussed about using `rust-esplora`  as our dep and not require separate api definition over `ureq`.. Is that still the plan?

Reply:
This should be rewritten to use `rust-esplora-client`.  I've already done that in the bdk PR https://github.com/bitcoindevkit/bdk/pull/793 so the changes here and there just need to be unified.